### ISSUE TITLE
[HighContrast][Edge][DropDown] High contrast issue on focus

### ISSUE THREAD

**Issue by ankitk94** [OfficeDev/office-ui-fabric-react#3846]
### Bug Report

- __Browser and OS versions__: Edge

#### Describe the issue:

When dropdown is focused, it turns border and text of same colour (Aqua) without any highlighting:

![image](https://user-images.githubusercontent.com/6390909/35624152-28fee88c-06b4-11e8-8968-a0cd875e1379.png)

Windows high contrast settings here expects to have either "White text on black background" (button text) or "black text with aqua background" (selected text):

![image](https://user-images.githubusercontent.com/6390909/35624467-0ef53c1a-06b5-11e8-9b88-70ad0de1e66e.png)

#### Actual behavior:

On focus, dropdown is showing aqua text on black background.

**Comment by betrue-final-final** [comment]
@lynamemi, we should talk about this one!

**Comment by betrue-final-final** [comment]
@lynamemi, here's what we talked about:

1. Reverse the colors of dropdown on focus
2. Double the thickness of the text field and combobox border on focus.
